src/main/java/com/onepagecrm/models/Company.java

```diff
@@ -86,7 +95,41 @@ public Company partialUpdate(Company updateValues) throws OnePageException {
         );
         Response response = request.send();
         String responseBody = response.getResponseBody();
-        return CompanySerializer.fromString(responseBody);
+        Company company = CompanySerializer.fromString(responseBody);
+        LoginSerializer.updateDynamicResources(responseBody);
+        return company;
+    }
+
+    public static ContactList getLinkedContacts(String companyId) throws OnePageException {
+        String endpoint = LINKED_CONTACTS_ENDPOINT.replace("{id}", companyId);
+        Request request = new GetRequest(endpoint, null);
+        Response response = request.send();
+        String responseBody = response.getResponseBody();
+        return LinkedContactsSerializer.fromString(responseBody);
+    }
```

Comment:
@lorobode Do you think it would be a good idea to add a non-static public method in this class which used the current instance id?

e.g.
```
public ContactList getLinkedContacts() throws OnePageException {
    getLinkedContacts(this.id);
}
```